### PR TITLE
to MyST-Parser & to pypi releases instead of git packages

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by odscjames** [comment]
This can be reviewed at http://docs.openreferral.org/en/2021-07-07-jb/

**Comment by odscjames** [comment]
Note https://github.com/openreferral/specification/issues/237 is still  
an issue; but it was an issue before this commit to so this commit has  
not changed this problem.

**Comment by Bjwebb** [comment]
I think recommonmark is still doing some of the work here, as its being pulled in by sphinxcontrib-opendataservices. Could you try it with branch, which removes that dependency https://github.com/OpenDataServices/sphinxcontrib-opendataservices/pull/32

File: docs/conf.py [hunk 1]
```diff
@@ -388,6 +387,10 @@ def _wrap(cell):
                 return para
             return [[_wrap(cell) for cell in row] for row in rows]
 
+        class DummyListTable():
```
Comment (robredpath) [inline docs/conf.py:390]: @odscjames the term "dummy" to mean "placeholder" or "stub" is something that's been highlighted as potentially problematic - alongside terms like "master". See, for example [Apple's style guide](https://help.apple.com/applestyleguide/#/apsg7af4f5d0) , which says :


> dummy
>     Don’t use to refer to placeholder content. Use an alternative that conveys the correct meaning in context, such as placeholder, sample, test, stub, or boilerplate.
> 
>     Correct: Use layout guides to replace the placeholder views you created.
> 
>     Incorrect: Use layout guides to replace the dummy views you created.

From how it's being used, I'm guessing this is something that's used as a base for other things? So maybe BaseListTable? Or InterimListTable?

Comment (odscjames) [reply docs/conf.py:390]: Fixed

**Comment by Bjwebb** [comment]
I've not had chance to review all of this yet, but I see that my previous comment is resolved. recommonmark is removed from the dependencies, and I see the change in formatting I would expect from the switch.

**Review by Bjwebb — APPROVED** [review]
The bullet points on http://docs.openreferral.org/en/2021-07-07-jb/hsds/ have odd spacing, but it's not a blocker.

**Review by Bjwebb — APPROVED** [review]

**Pull request merged**